File: LeetCode Solutions/Problems/2455. Average Value of Even Numbers That Are Divisible by Three.py
class Solution:
    def averageValue(self, nums: List[int]) -> int:
        result, count = 0, 0
        for i in range(len(nums)):
            if nums[i] % 2 == 0 and nums[i] % 3 == 0:
                result += nums[i]
                count += 1
        if count != 0:
            return result//count
        return 0

File: LeetCode Solutions/Problems/2455. Average Value of Even Numbers That Are Divisible by Three.java
class Solution {
    public int averageValue(int[] nums) {
        int result = 0, count = 0;
        for (int i=0; i<nums.length; i++){
            if (nums[i] % 2 == 0 && nums[i] % 3 == 0){
                result += nums[i];
                count += 1;
            }
        }
        if (count != 0){
            return result/count;
        }
        return 0;
    }
}